src/models/autocompleteservice/AutocompleteResponse.ts

```diff
@@ -24,29 +25,41 @@ export interface AutocompleteResponse {
  * @public
  */
 export interface FilterSearchResponse {
-  /** Indicates that the results are separated by field in the sections property. */
-  sectioned: boolean,
   /**
    * Represents autocomplete results separated by field.
    *
    * @remarks
-   * This array will only be populated if sectioned is true.
+   * If sectioned is true, the matching filters will be returned in a separate section per field.
+   * By default, they are all returned in the same section.
    */
   sections: {
-    /** A display label for the field. */
-    label: string,
+    /** A display label for the field.
+     *
+     * @remarks
+     * When sectioned is false, there's no label since all filters wil be returned in same section.
+     */
+    label?: string,
     /** An array of {@link AutocompleteResult}s. */
     results: AutocompleteResult[];
   }[],
-  /**
-   * An array of {@link AutocompleteResult}s.
-   *
-   * @remarks
-   * This array will only be populated if sectioned is false.
-   */
-  results: AutocompleteResult[],
-  /** {@inheritDoc SearchIntent} */
-  inputIntents: SearchIntent[];
+  /** Contains error information when one or more verticals fail to return results. */
+  failedVerticals: {
```

Comment:
@mgdunn2 do you think this is something worth exposing to a user of Core? The concept of a `failedVertical` for filter search seems kind of funny. It makes sense for something like universal search. Is this deprecated?

Reply:
At the moment if there is an error in filter search it will be returned as a failed vertical. Since filter search is now being scoped to only a single vertical it might make sense to just fail the request entirely rather than return a 200 with a failed vertical but this does reflect the current state of the API.